homework/App.js

```diff
@@ -52,16 +74,14 @@ class App {
       const container = document.getElementById('container');
       App.clearContainer(container);
 
-      const leftDiv = Util.createAndAppend('div', container);
-      const rightDiv = Util.createAndAppend('div', container);
-
-      const contributorList = Util.createAndAppend('ul', rightDiv);
+      repo.render(container);
+      const rightDiv = Util.createAndAppend('div', container, { id: 'contsDiv' });
 
-      repo.render(leftDiv);
+      Util.createAndAppend('p', rightDiv, { id: 'cont-header', text: 'Contributors' });
 
       contributors
         .map(contributor => new Contributor(contributor))
-        .forEach(contributor => contributor.render(contributorList));
+        .forEach(contributor => contributor.render(rightDiv));
```

Comment:
Follow to the next comment, but keep in mind, this line is executed repeatedly as long as we have a contributors.